backend/vcdat/app.py

```diff
@@ -18,5 +20,11 @@ def serve_resource_file(path):
         print 'Unable to serve file ', path
         return send_from_directory(dir_path, 'taco')
 
+@app.route("/getTemplates")
+def get_templates():
+    templates = sorted(vcs.elements['template'].keys())
+    print templates
```

Comment:
I removed them all from the next pull request. I also need to fix the conflicts with master.